src/main/java/io/codelex/loops/practice/RollTwoDice.java

```diff
@@ -0,0 +1,34 @@
+package io.codelex.loops.practice;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class RollTwoDice {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter desired sum? (2-12)");
+
+        int desiredSum = input.nextInt();
+        while (desiredSum < 2 || desiredSum > 12) {
+            System.out.println("Incorrect number");
+            desiredSum = input.nextInt();
+        }
+
+        Random rand = new Random();
+        int a = rand.nextInt(6);
```

Comment:
Labāk būtu to 1 uzreiz pieskaitīt šādi: a = rand.nextInt(6) + 1;
Tādā veidā neveltot papildus koda rindu tam. (Tas pats attiecas uz mainīgo b)